crypto/evp/e_aes.c

```diff
@@ -1349,9 +1350,13 @@ static int aes_gcm_ctrl(EVP_CIPHER_CTX *c, int type, int arg, void *ptr)
             return 0;
         if (arg)
             memcpy(gctx->iv, ptr, arg);
-        if (EVP_CIPHER_CTX_encrypting(c)
-            && RAND_bytes(gctx->iv + arg, gctx->ivlen - arg) <= 0)
-            return 0;
+        if (EVP_CIPHER_CTX_encrypting(c)) {
+            if (c->drbg != NULL) {
+                if (RAND_DRBG_generate(c->drbg, gctx->iv + arg, gctx->ivlen - arg, 0, NULL, 0) <= 0)
```

Comment:
You could replace RAND_DRBG_generate() by RAND_DRBG_bytes(), now that #4752 has been merged.